components/script/dom/webglprogram.rs

```diff
@@ -574,6 +574,11 @@ impl WebGLProgram {
             return Err(WebGLError::InvalidValue);
         }
 
+        let mut active_uniforms = self.active_uniforms.borrow_mut();
+        if active_uniforms.len() > block_binding as usize {
+            active_uniforms[block_binding as usize].bind_index = Some(block_binding);
```

Comment:
Is there any point when we should reset this to None?

Reply:
I'm not sure, but I didn't find any circumstance when we should set this to None except the initial state.